### PR TITLE
Add `aliases` option

### PR DESCRIPTION
**Pull request by tommy-mitchell**
Adds `flag.aliases` as specified in #109:

```js
flags: {
	unicorn: {
		type: 'string',
		shortFlag: 'u',
		aliases: ['unicorns']
	}
}
//=> Accepts `--unicorn`, `--unicorns`, or `-u`
```

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -97,6 +97,15 @@ const buildParserFlags = ({flags, booleanDefault}) => {
...
 		}
 
+		if (Array.isArray(flag.aliases)) {
+			if (flag.alias) {
+				flag.aliases.push(flag.alias);
+			}
+
+			flag.alias = flag.aliases;
+			delete flag.aliases;
+		}
```
Comment (tommy-mitchell) [inline index.js:107]: This will also allow an empty array:

```js
flags: {
	unicorn: {
		type: 'string',
		aliases: []
	}
}
```

Should this be an error?

Comment (sindresorhus) [reply index.js:107]: > Should this be an error?

No

**Comment by sindresorhus** [comment]
@tommy-mitchell Do you plan any more changes? If not, I will do a new version.

**Comment by tommy-mitchell** [comment]
I was also working on a `.choices` option.

**Pull request merged**